### ISSUE TITLE
configurable timeout on invitation to server

### ISSUE THREAD

**Issue by midget3111** [trigg/Rebuttal#42]

**Comment by trigg** [comment]
Is this needed? invites are currently unique so expire after use

**Comment by midget3111** [comment]
Yes. Some communities will 100% want an invite that doesn't expire after use.

**Comment by trigg** [comment]
Oh you don't mean you want invitations that time out after a time limit

**Comment by trigg** [comment]
We've got non-expiring invite as an option.

**Comment by midget3111** [comment]
Config option is great, but would be useful for this to be in the GUI server settings menu as well.

**Comment by trigg** [comment]
Add indicator pointing to infinite signup url when enabled
